unstructured/partition/pdf.py

```diff
@@ -548,40 +577,32 @@ def add_pytesseract_bbox_to_elements(elements, bboxes, width, height):
         height=height,
     )
 
-    boxes = bboxes.strip().split("\n")
-    i = 0
+    boxes = bboxes_string.strip().split("\n")
+    box_idx = 0
     for element in elements:
+        if not element.text or boxes[box_idx][0]:
```

Comment:
Previous comment was supposed to be attached to this line.

Reply:
Honesty didn't check to see what object was making this fail (guessing it is None), but I'll go through and see and come up with a better check. I was just getting it to not fail any more.

Reply:
The index error was just coming from the while loop (presumably if it was at the end of the element text). I put it in a try/except to catch this, but let me know if there's a different preferred method.